packages/material-ui/src/useAutocomplete/useAutocomplete.js

```diff
@@ -493,6 +493,24 @@ export default function useAutocomplete(props) {
     syncHighlightedIndex();
   });
 
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    React.useEffect(() => {
+      if (!inputRef.current || inputRef.current.nodeName !== 'INPUT') {
+        console.error(
+          [
+            `Material-UI: The input ref is not binded correctly, it resolves to: ${inputRef.current}.`,
```

Comment:
```suggestion
            `Material-UI: Unable to find the input element. It was resolved to ${inputRef.current} while an HTMLInputElement was expected.`,
```

Assuming `inputRef` is the prop name.

I don't think we use "ref binding" anywhere else.